i18ng.js

```diff
@@ -64,7 +88,7 @@ angular.module('i18ng')
 
         var translations = {}
         var t = translate.bind(null, scope, element, translations)
-        var ignore = ['opts', 'html']
+        var ignore = ['opts', 'html', 'nestedhtml']
```

Comment:
probably couldn't hurt, but it only cares if the tag also has the `i18ng` directive on it